requirements.txt

```diff
@@ -12,3 +12,9 @@ pillow-heif==0.16.0
 chardet==5.2.0
 structlog==24.4.0
 SQLAlchemy==2.0.34
+pydub==0.25.1
+mutagen==1.47.0
+numpy==1.26.4
+ffmpeg-python==0.2.0
+Send2Trash==1.8.3
```

Comment:
this dependency looks unused

Reply:
Send2Trash was accidentally added due to its inclusion in future a v9.4 PR, fixed 👍